pipelines/mix/tasks/get_mixed.py

```diff
@@ -151,24 +198,55 @@ def get_mixed_output(self, mix_chart):
                     initialization = True
 
             if self.output_normalization_flag:
-                mixed_df = self.normalize(mixed_df)
+                mixed_df = self.normalize(mixed_df, self.summer_normalization)
+
+            mixed_hourly_normalized = self.normalize_hourly(mixed_df)
             
             mixed_df.insert(loc=0, column='target', value=location)
             mixed_df.insert(loc=1, column='daytype', value=df.daytype)
             mixed_df.insert(loc=2, column='time', value=df.time)
+
+            mixed_hourly_normalized.insert(loc=0, column='target', value=location)
+            mixed_hourly_normalized.insert(loc=1, column='daytype', value=df.daytype)
+            mixed_hourly_normalized.insert(loc=2, column='time', value=df.time)
+
             output_df = output_df.append(mixed_df)
+            output_df_hourly_normalized = output_df_hourly_normalized.append(mixed_hourly_normalized)
 
-        return output_df
+        return output_df, output_df_hourly_normalized
 
-    def normalize(self, mixed_df):
+    def normalize(self, mixed_df, summer_normalization):
+        """
+        if summer_normalization == True it will be normalized to peak summer
+        """
         min_val = mixed_df.min().min()
         mixed_df = mixed_df - min_val
 
-        max_val = mixed_df.sum(axis=1).max()
+        if summer_normalization:
+            max_val = mixed_df[48:72].sum(axis=1).max()
+        else:
+            max_val = mixed_df.sum(axis=1).max()
+
         mixed_df = mixed_df/max_val
 
         return mixed_df
 
+    def normalize_hourly(self, mixed_df):
+        mixed_hourly_normalzied = mixed_df.div(mixed_df.sum(axis=1), axis=0)
+
+        return mixed_hourly_normalzied
+
+    def get_study_period(self, full_df):
+        daytypes = self.study_periods.keys()
+        study_hours_df = pd.DataFrame(columns=full_df.columns)
+
+        for daytype in daytypes:
+            for time in self.study_periods[daytype]:
+                day_df = full_df.loc[(full_df['daytype'] == daytype) & (full_df['time'] == time)]
+                study_hours_df = study_hours_df.append(day_df)
+
+        return study_hours_df
```

Comment:
I think there's a little confusion here on the the `self.study_periods` dictionary setup and the looping here.

All of this can be accomplished simply, e.g.:
(don't copy and paste this syntax, but you get the gist)
```
study_hours_df = pd.DataFrame(columns=full_df.columns)

for key, value in self.study_periods.items():
        day_df = full_df.loc[(full_df['daytype'] == key) & (full_df['time'] == value)]
        study_hours_df = study_hours_df.append(day_df)
```

Reply:
fixed